### PR TITLE
fix: bump sdk and utils version

### PR DESCRIPTION
**Pull request by preethamr**
## Type of change

- [ ] Docs change / dependency upgrade
- [x] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/utils/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@connext/nxtp-utils",
-  "version": "2.5.0-alpha.1",
```
Comment (LayneHaber) [inline packages/utils/package.json:3]: are we skipping a version here?

**Review by LayneHaber — COMMENTED** [review]
changelog?

File: packages/utils/package.json (hunk 1 above)
Comment (preethamr) [reply packages/utils/package.json:3]: Yes.
Syncing with sub version numbers to be the same with the other packages

**Comment by preethamr** [comment]
> changelog?

Added change log

**Review by LayneHaber — APPROVED** [review]

**Pull request merged**